lib/Doctrine/Annotations/Metadata/Exception/TooManyDefaultProperties.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Annotations\Metadata\Exception;
+
+use LogicException;
+
+final class TooManyDefaultProperties extends LogicException
+{
+    public static function new() : self
+    {
+        return new self('The annotation can only have at most one default property.');
```

Comment:
This exception message should at least contain the name of the annotation.

It would probably also be helpful if it listed the properties that are currently marked as default - this would help users find the cause of the error.